### ISSUE TITLE
Phalcon\Mvc\Model\Exception: Column "DB_ROWNUM" doesn't make part of the column map

### ISSUE THREAD

**Issue by dancebear** [phalcon/cphalcon#3256]
<?php
	class Order extends \Phalcon\Mvc\Model {

	    /**
	     * 预约表的ID
	     * @Primary
	     * @Identity
	     * @Column(type="integer",nullable=false,column="YUYUE_ID")
	     * @var int
	     */
	    public $orderId;
	    
	    /**
	     * 订单编号
	     * @Column(type="integer",nullable=false,column="ORDER_NO")
	     * @var int
	     */
	    public $orderNo;
	   
	}
?>

Order::find();will throw a exception : Phalcon\Mvc\Model\Exception: Column "DB_ROWNUM" doesn't make part of the column map
Order::count();will throw a exception:Phalcon\Mvc\Model\Exception: Column "ROWCOUNT" doesn't make part of the column map

i use PDO\Oracle driver

**Comment by andresgutierrez** [comment]
Can you please try again using Phalcon 2?

**Comment by andresgutierrez** [comment]
This is fixed in the 2.0.x branch (Phalcon 2.0.4), you have to change a global ORM setting to allow unexpected columns:

```php
\Phalcon\Mvc\Model::setup(['ignoreUnknownColumns' => true]);
```

**Comment by rlaffers** [comment]
Please reopen this bug. The referenced commit fixes it only partially. Even with ignoreUnknownColumns => true the exception is thrown when using columnMap() and I try to find() with the *limit* option and then calling toArray() on the result:

```php
//...trimmed model Languages
public function columnMap() {
    return array(
        'ID_LANGUAGE' => 'id',
        'TITLE' => 'title',
        'CODE_2' => 'code_2',
        'CODE_3' => 'code_3',
        'LOCALE' => 'locale'
    );
}
// ...later in the code
$result = Languages::find(array('limit' => 2));      
echo '<pre>' . var_export($result->toArray(), 1) . '</pre>';  // throws Phalcon\Mvc\Model\Exception: Column 'PHALCON_RN' is not part of the column map
```
The DB used is Oracle 10g. Phalcon 2.0.8. Connection configuration is as follows:
```php
    'database' => array(
        'adapter'     => 'Oracle',
        'host'        => 'orahost',
        'username'    => 'webuser1',
        'password'    => 'secretpswd',
        'dbname'      => 'monitor',
        'schema'      => 'COMMON',
        'options' => array(
            \PDO::ATTR_CASE => \PDO::CASE_UPPER
        ),
    ),
```
There are 2 workarounds:
- If I don't specify columnMap() in the model, the exception goes away. That is not an option however, as I need to map column names which are UPPERCASE to lowercase model properties.
- If I add 
```php
            'PHALCON_RN' => '__foo__',
```
to the columnMap then the exception goes away. These are just workarounds though, not acceptable solutions.
